sewer/dns_providers/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .cloudflare import CloudFlareDns  # noqa: F401
 from .acmedns import AcmeDnsDns  # noqa: F401
 from .aliyundns import AliyunDNS  # noqa: F401
+from .hedns import HEDNS  # noqa: F401
```

Comment:
@menduo The other classes are called `CloudFlareDns` `AcmeDnsDns` 
so I think this should also be called `HeDns` so that we can have consistency. 

We should probably rename `AliyunDNS` to `AliyunDns` for the same reasons; but I do not know if that will break compatibility for you. If it would break compatibility, maybe we can do it in another Pull request rather than this one

Reply:
OK.